llvm/include/llvm/Transforms/Utils/SampleProfileInference.h

```diff
@@ -57,6 +58,7 @@ struct FlowFunction {
   std::vector<FlowJump> Jumps;
   /// The index of the entry block.
   uint64_t Entry{0};
+  uint64_t Sink{0};
```

Comment:
Nit: add a description and initialize to `uint64_t` max value to indicate that a proper initialization will happen later.